tests/rustdoc-gui/src/test_docs/lib.rs

```diff
@@ -652,3 +652,20 @@ pub mod long_list {
     //! * [`FromBytes`](#a) indicates that a type may safely be converted from an arbitrary byte
     //!   sequence
 }
+
+pub struct ImplDoc;
+
+/// bla sondfosdnf sdfasd fadsd fdsa f ads fad sf sad f sad fasdfsafsa df dsafasdasd fsa dfadsfasd
+/// fads fadfadd
```

Comment:
Question/discussion:

- What happens if this "first summary line" is quite long? Does it abruptly cut off?
- Should this have some kind of trailing "..." to indicate that it has more (hidden) content instead of that is the only available content?

Reply:
You have the visual effect you can see in the first screenshot which allows you to see that there is more content. So the first paragraph is kept entirely, untouched. Just partially hidden if too long when impl block is collapsed.

Reply:
Forgot to precise: you can see it in action by reducing the width of the window so the content ends up on two lines (like in the screenshot).

Reply:
Ah yeah, it was a bit subtle in dark theme which is why I thought it was just cut off.

Reply:
The idea was to have as little changes as possible in the DOM so I just limited the element size when the item is collapsed. Anyway, now you know everything. ;)